Workbooks/Windows Virtual Desktop/Connection Performance/Connection Performance.workbook

```diff
@@ -942,9 +942,9 @@
                     ],
                     "showExportToExcel": true,
                     "queryType": 0,
-                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "resourceType": "microsoft.desktopvirtualization/hostpools",
                     "crossComponentResources": [
-                      "{Workspace}"
+                      "{HostPool}"
```

Comment:
All the other queries on the scoped worbook are using the LA query instead of host pool queries. For consistent, please,
use {poolla} and filter by resourceID
              "resourceType": "microsoft.operationalinsights/workspaces",
              "crossComponentResources": [
                "{poolla}"

Reply:
Changed the resourceType back to microsoft.operationalinsights/workspaces and crossComponentResources to {poolla}.